packages/notifi-dapp-example/src/components/DestinationPanel.tsx

```diff
@@ -65,8 +66,9 @@ export const DestinationPanel: React.FC<DestinationPanelProps> = ({
   const [isLoading, setIsLoading] = useState(false);
   const { wallets, selectedWallet } = useWallets();
   const { popGlobalInfoModal } = useGlobalStateContext();
+  const xmtp = useClient();
 
-  const signWallet = async () => {
+  const xip43Impl = async () => {
```

Comment:
Not sure if this one still being used? It seems not being used anywhere and duplicated with the `xmtpXip42Impl` method

Reply:
We want to keep this because it will be used soon

Reply:
This will be a fast follow

Reply:
Temporarily commenting out this function because it will be needed later